components/usage/pkg/apiv1/usage.go

```diff
@@ -136,6 +136,25 @@ func (s *UsageService) ReconcileUsage(ctx context.Context, req *v1.ReconcileUsag
 
 }
 
+func (s *UsageService) GetCostCenter(ctx context.Context, in *v1.GetCostCenterRequest) (*v1.GetCostCenterResponse, error) {
+	result, err := db.GetCostCenter(ctx, s.conn, db.AttributionID(in.AttributionId))
+	if err != nil {
+		return nil, status.Errorf(codes.NotFound, "Cost center not found: %s", err)
+	}
+
+	attributionID, err := db.ParseAttributionID(in.AttributionId)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to parse attribution ID: %w", err)
+	}
+
+	return &v1.GetCostCenterResponse{
+		CostCenter: &v1.CostCenter{
+			AttributionId: string(attributionID),
```

Comment:
Weirdly I still have to cast it to a string here. Otherwise I get `cannot use attributionID (variable of type db.AttributionID) as string value in struct literal`